src/components/shared/tests/SearchInput.spec.js

```diff
@@ -4,27 +4,32 @@ import { shallow } from "enzyme";
 import SearchInput from "../SearchInput";
 
 describe("SearchInput", () => {
+  // !! wrapper is defined outside test scope
+  // so it will keep values between tests
   const wrapper = shallow(
     <SearchInput
       query=""
-      count={0}
+      count={5}
       placeholder="A placeholder"
       summaryMsg="So many results"
-      showErrorEmoji
+      showErrorEmoji={false}
     />
   );
-  it("render", () => expect(wrapper).toMatchSnapshot());
-  it("show svg (emoji)", () => {
-    wrapper.setProps({ query: "test" });
+
+  it("renders", () => expect(wrapper).toMatchSnapshot());
+
+  it("shows svg error emoji", () => {
+    wrapper.setProps({ showErrorEmoji: true });
```

Comment:
this is testing the behavior by just changing the emoji... however this change should reflect the count.. perhaps using the count was a better way of verifying the behavior?

A nice way to think of tests is to ask these two questions:

what is the expected input?
what is the expected output?

the input isn't really the showEmoji flag, its the number of results that come back.

Maybe a test to return to!

Reply:
I'd say this is definitely a test to return to. I just had enough time to play around with it and see how it functions. Now comes the second part is rewriting it so that it actually tests something meaningful.